.vscode/settings.json

```diff
@@ -7,8 +7,12 @@
     "**/CVS": true,
     "**/.DS_Store": true,
     ".nyc_output": true,
-    "coverage": true,
-    "packages/**/lib": true,
-    "packages/**/lib-test": true
+    "coverage": true
+  },
+  "search.exclude": {
+    "**/node_modules": true,
+    "**/bower_components": true,
+    "**/lib": true,
+    "**/lib-test": true
```

Comment:
Let me know if you can't bear these changes. Personally i find keeping the generated code out of the search results is more important that keeping it out of the IDE, I like to poke around in it occasionally.